docs/examples.rst

```diff
@@ -203,5 +203,5 @@ as indicated in the profile (JSON schema) of the record of interest. The Portal
 reject or silently ignore any attempt to remove such properties, nonetheless, to be a good citizen,
 this client performs these checks regardless for good measure.
 
-It should also be noted that some properties simply can't be deleted.  For example, any attempt
+It should also be noted that some properties simply can't be deleted. For example, any attempt
```

Comment:
that is true only if alias has default and allows empty list, which it doesnt in IGVF - so we need a different example for IGVF